### PR TITLE
Authentication

### PR DESCRIPTION
**Pull request by vrx29**
Login and Sign Up pages added.

Live https://gimesto-store.netlify.app/pages/signup.html

### CODE REVIEW

**Review by anik31 — APPROVED** [review]
Nice UI & clean code. You can merge.

**Review by 05him — APPROVED** [review]
image
UI and code looks good just one minute suggestion that you should try using rem or % instead of px

File: styles/utilities/anchor.css [hunk 1]
```diff
@@ -3,4 +3,5 @@ a {
   display: flex;
```
Comment (MayankKumar10) [inline styles/utilities/anchor.css]: use helper class for flex

File: styles/components/authentication.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
+.auth-cont {
+  width: 450px;
```
Comment (MayankKumar10) [inline styles/components/authentication.css]: use rem or % in width

File: styles/components/authentication.css [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  border-radius: 0.5rem;
+  margin: 2rem auto;
+  padding: 2rem;
+  box-shadow: 0px 0px 25px 10px #f8f8fb;
```
Comment (MayankKumar10) [inline styles/components/authentication.css:6]: if you use this box shadow anywhere else make it a helper class so you do have to repeat it.

**Review by MayankKumar10 — APPROVED** [review]
Everything else looks good to me

File: styles/components/authentication.css (hunk 3 above)
Comment (vrx29) [reply styles/components/authentication.css:6]: This is just for this particular card and I dont want to change it when the user change the font size of the site. So to make it fixed I havae used it in px not in rem

**Pull request merged**